packages/react-reconciler/src/ReactFiberReconciler.new.js

```diff
@@ -714,6 +717,11 @@ export function getIsStrictModeForDevtools() {
 }
 
 export function setIsStrictModeForDevtools(newIsStrictMode: boolean) {
+  // We're in a test. Disable yielding during the double render
+  if (typeof Scheduler.unstable_yieldValue === 'function') {
+    Scheduler.unstable_setDisableYieldValue(newIsStrictMode);
+    setSuppressWarning(newIsStrictMode);
```

Comment:
so when we set strict mode to true, we disable the test yields and also any warnings during this double render, but only for tests, right?

Reply:
Yup! `Scheduler.unstable_yieldValue` is a `SchedulerMock` only function, which only exists for tests.